service/src/main/resources/application.yml

```diff
@@ -25,12 +25,12 @@ gpc-consumer:
     rootCA: ${GPC_CONSUMER_SPINE_ROOT_CA_CERT:}
     subCA: ${GPC_CONSUMER_SPINE_SUB_CA_CERT:}
     gpcConsumerUrl: ${GPC_CONSUMER_URL:http://localhost:8090}
-    gpcUrl: ${GPC_CONSUMER_GPC_GET_URL:}
+    overrideGpcProviderUrl: ${GPC_CONSUMER_OVERRIDE_GPC_PROVIDER_URL:}
```

Comment:
will we need separate gp2gp docker-compose pr for this variable update **GPC_CONSUMER_OVERRIDE_GPC_PROVIDER_URL**  and one below **GPC_CONSUMER_SSP_URL**?